README.md

```diff
@@ -9,36 +16,42 @@ npm     | >= 4.5x
 
 # Clone the repository
 
-```shell=
-git clone git@github.com:Lugdunum3D/LugBench-API.git
+Download the project [here](https://github.com/Lugdunum3D/LugBench-API/archive/dev.zip)
```

Comment:
I was talking about cloning by https not downloading the project.
`git clone https://github.com/Lugdunum3D/LugBench-API.git`